Doc/library/calendar.rst

```diff
@@ -185,48 +185,51 @@ it's the base calendar for all computations.
 
       Note that the length of this list must be seven items.
 
-   .. attribute:: noday_classes
+   .. attribute:: noday_class
 
       The CSS class for a weekday occurring in the previous or coming month.
 
    .. attribute:: weekday_head_classes
 
-      A list of CSS styles used for each weekday. The default style list is
-      the same as :attr:`cssclasses`.
+      A list of CSS classes used for weekday names in the header row.
+      The default the same as :attr:`cssclasses`.
 
-   .. attribute:: month_head_classes
+   .. attribute:: month_head_class
 
-      A space separated list of styles for the month head, for example::
+      The month's head CSS class.
+      The default value is ``"month"``.
 
-      "text-bold text-red"
 
-      The default value is ``"month"``.
+   .. attribute:: month_class
 
-   .. attribute:: month_classes
+      The CSS class for the whole month's table.
 
-      A space separated list of styles for the whole table.
       The default value is ``"month"``.
 
-   .. attribute:: year_classes
+   .. attribute:: year_class
 
-      A space separated list of styles for the table when formatting the year
-      as a table of tables (see: :meth:`formatyear`).
+      The CSS class for the whole year's table of tables (see: :meth:`formatyear`).
       The default value is ``"year"``.
 
-   .. attribute:: year_head_classes
+   .. attribute:: year_head_class
 
-      A space separated list of styles for the table head when formatting the
-      year as a table of tables (see: :meth:`formatyear`).
+      The CSS class for the whole year's table of tables (see: :meth:`formatyear`).
       The default value is ``"year"``.
```

Comment:
This and the previous description are the same. They should be distinct, so that the user can determine for whihc part of the page the class gets used.